walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -23,28 +23,23 @@
 
 public final class FindMeetingQuery {
   public Collection<TimeRange> query(Collection<Event> events, MeetingRequest request) {
-    // The Collection that stores the available times for the requested meeting.
-    Collection<TimeRange> availableTimes = new ArrayList<TimeRange>();
-
     // Check to see if the duration is longer than a day.
     if (request.getDuration() == TimeRange.WHOLE_DAY.duration() + 1) {
-      return availableTimes;
+      Collection<TimeRange> availableTimes = new ArrayList<TimeRange>();
+      return  availableTimes;
```

Comment:
You can do:
return ImmutableSet.of();

Reply:
What library is this in? Because when I try importing com.google.common.collect.ImmutableSet, that doesn't seem to work. Also, do you want want me to just get rid of line 28 and just make this if statement return ImmutableSet.of()?

Reply:
https://guava.dev/releases/23.0/api/docs/com/google/common/collect/ImmutableSet.html
Yes. You can get rid of line 28 but just return an empty set.

Reply:
So I tried importing the library for this but it still didn't seem to work? 
import com.google.common.collect.ImmutableSet;
This gave me an error saying this package doesn't exist. Is this not the right thing I need to import?

Reply:
Looks like it takes some efforts to use those libraries (guava) in gcloud. For example:
https://stackoverflow.com/questions/32242229/compiling-google-cloud-service-client-library-with-guava
Let's not do this for now (or you can still do "return new ArraList..." instead of creating a variable) -- but we will have to figure it out at some point.

Reply:
Okay, sounds good. I made that change